lib/web_ui/lib/src/engine/view_embedder/embedding_strategy/embedding_strategy.dart

```diff
@@ -26,16 +26,12 @@ abstract class EmbeddingStrategy {
     }
   }
 
+  /// The host element in which the Flutter view is embedded.
+  DomElement get hostElement;
```

Comment:
Is this embedding strategy getting too empty to warrant it to be 3 classes?

Reply:
Yep, I might move them all into a single file later.